staging/src/github.com/kubeedge/beehive/pkg/common/type.go

```diff
@@ -0,0 +1,16 @@
+package common
+
+// define channel type
+const (
+	// MsgCtxTypeChannel message type channel
+	MsgCtxTypeChannel = "channel"
+	// MsgCtxTypeSocket message type socket
+	MsgCtxTypeSocket  = "tcp"
+	// MsgCtxTypeUS message type us
+	MsgCtxTypeUS = "unixpacket"
```

Comment:
These two types are all used now?

Reply:
yes

Reply:
Oh, i'll keep the `"unixpacket"` and remove the `"tcp"`

Reply:
done